news/2024/2024-05-20-the-followpoint-yesmydarknesss-the-osu-catch-prodigy.md

```diff
@@ -0,0 +1,95 @@
+
+*Why did you choose to play osu!catch of all gamemodes osu! has to offer?*
+
+> As I said in a previous answer, first time I saw osu! was ExGon's score, and that's when I started to play osu!. That's one of the main reasons I continued playing catch, as well as it being easy to start (but ducking impossible to master). So after I found osu!catch, I started to grind it a lot. I think my goal at this moment was to become a superior player like ExGon.
+
+*What do you think of the osu!catch and its community in its current state?*
+
+> Sadly, it's a dying gamemode, there isn't much to attract a lot of players. It's easy to join, but hard to understand the logic. Looking from the outside, most of people are saying that catch looks "super easy" and then don't even try, because it "looks easy". Not many understand that this gamemode gets harder and harder as you play more.
+>
+> Our community is quite small, because not many new players join, so sadly it's mostly the same people for years. I'd recommend people to actually try playing catch, maybe you'll like it and become the next star!
+
+*Any advices you would give to people who want to or just started playing osu!catch?*
+
+> A lot of people as me that actually. Something I'd like to say first is, there is no easy way to become better at rhythm games. Most people don't even have a playcount close to my [Calamity Fortune](https://osu.ppy.sh/beatmapsets/114741#fruits/344892) play retries (which is around 10,000), which I believe is too early to understand the gamemode properly. After getting close to that playcount you may finally start to understand what the game is and how it works fully, not only in catch, but in other rhythm games too.
+>
+> Next, I'd like to talk about how to practice. In rhythm games, there aren't many tips on how to improve. I think tips are more about how to hit special patterns, not something that applies to the whole game. If a "tip" applies to the entire game, then it's not really a tip anymore; it's skill.
+>
+> So, what I want to say is try to figure out the best practice method for you. In my case, I was spamming retries a lot, playing each hard map more than a thousand times. But even while spamming retries, you need to think about the reasons for your gameplay, like 'Why did I miss that pattern?' or 'Do I need to change my beat to be faster or slower?' or 'I need to memorize the position of the fruit in this part.'
+>
+> Yes, after you move past the spamming phase, you need to find your own reasons for your gameplay. Why did you miss? How do you catch the patterns? What beat should you use?
+> In my case, when I missed a section, I kept trying to change my beat. If I was tapping my keyboard 1/4 at that moment, I would switch to 1/3 or 1/6, constantly adjusting my timing. Once I found the right beat, I memorized it and kept it in mind when retrying that map.
```

Comment:
i have no idea what this is supposed to be about. identifying what beat snap divisor is used in a map? adjusting by tapping earlier/later? it just doesn't make any sense

should check with yesmydarkness

Reply:
```suggestion
> In my case, when I miss a section, I keep trying to change my tapping rhythm. If I were tapping my keyboard at 1/4 in one moment, I would switch to 1/3 or 1/6, constantly adjusting my tapping. Once I found the right rhythm, I memorized it and kept it in mind when retrying the map.
```